src/main/java/com/minecraftabnormals/endergetic/core/events/CompatEvents.java

```diff
@@ -22,7 +22,7 @@
 	public static void onPotionExpire(PotionEvent.PotionExpiryEvent event) {
 		LivingEntity affected = event.getEntityLiving();
 		boolean isBabyEffect = event.getPotionEffect().getPotion() == ForgeRegistries.POTIONS.getValue(new ResourceLocation("savageandravage:shrinking"));
-		if (isBabyEffect || event.getPotionEffect().getPotion() == ForgeRegistries.POTIONS.getValue(new ResourceLocation("savageandravage:growth"))) {
+		if (isBabyEffect || event.getPotionEffect().getPotion() == ForgeRegistries.POTIONS.getValue(new ResourceLocation("savageandravage:growing"))) {
```

Comment:
Perhaps the potions should be cached here? (i.e. caching `ForgeRegistries.POTIONS.getValue(new ResourceLocation()`)
Or at least have the resource locations be constants.